pkg/manager/diff.go

```diff
@@ -585,15 +585,21 @@ func (rr *reproRunner) Run(ctx context.Context, r *repro.Result) {
 	ret := reproRunnerResult{origReport: r.Report, repro: r}
 
 	var result *instance.RunResult
-	var err error
-	for i := 0; i < 3; i++ {
+	var doneRuns int
+
+	for doneRuns < 3 {
+		if ctx.Err() != nil {
+			return
+		}
+
 		opts := r.Opts
 		opts.Repeat = true
-		if i == 0 || i == 1 {
+		if doneRuns < 2 {
 			// Two times out of 3, test with Threaded=true.
 			// The third time we leave it as is in case it was important.
```

Comment:
By "leaving it as is" do you mean setting to false, or using some test-specific default?

Reply:
`Opts` are coming from `pkg/repro` together with the reproducer program, so essentially we overwrite it the first two times and don't touch it the third time.

Updated the comment.